CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 5.0.0-beta.2 (2021-09-02)
+
+- Enabled Ivy's "partial" compilation mode. [#22](https://github.com/blackbaud/skyux-a11y/pull/22)
```

Comment:
```suggestion
- Enabled Ivy's `partial` compilation mode. [#22](https://github.com/blackbaud/skyux-a11y/pull/22)
```
I don't think this should be in quote marks here even though it is in the code, And since it's in the code, I'd make it a code snippet. ... But I'll approve the PR as is because I don't think it's a huge deal either way. Just let me know if you make the change and need me to re-approve.